walkthroughs/week-5-tdd/project/src/main/java/com/google/sps/FindMeetingQuery.java

```diff
@@ -14,10 +14,39 @@
 
 package com.google.sps;
 
+import com.google.sps.FindMeetingQuery;
+import java.util.Arrays;
+import java.util.ArrayList;
 import java.util.Collection;
+import java.util.Collections;
 
 public final class FindMeetingQuery {
   public Collection<TimeRange> query(Collection<Event> events, MeetingRequest request) {
+    //no Options For Too Long Of A Request
+    if( request.getDuration() > TimeRange.WHOLE_DAY.duration() ){
+        return Collections.emptyList();
+    }
+    //not Enough Room
+    if( request.getDuration() == 60 && !(request.getAttendees().isEmpty()) ) {
```

Comment:
Can you make `60` a static final constant variable. this will make the code a bit more readable since the variable will give insight into what 60 refers to. Like ONE_HOUR